sys/Makefile.dep

```diff
@@ -331,6 +331,9 @@ ifneq (,$(filter gnrc_sixlowpan_frag_sfr,$(USEMODULE)))
   USEMODULE += gnrc_sixlowpan_frag_rb
   USEMODULE += evtimer
   USEMODULE += xtimer
+  ifneq (,$(filter gnrc_tx_sync,$(USEMODULE)))
+    $(error module gnrc_tx_sync conflicts with gnrc_sixlowpand_frag_sfr)
+  endif
```

Comment:
:-/ I did not read the whole discussion on 6LoWPAN, but could you shortly summarize why? I use `gnrc_neterr` there and with #15695 in mind I would prefer if I could keep it that way.

Reply:
It is just not implemented and covered by the test yet. I planned to do this as a follow up, but I can also do this also in this PR.

Reply:
I'm fine with doing it as a follow-up :-).

Reply:
I rephrase the commit message a bit and added a TODO comment here.